ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/EVPN-MULTICAST-L3LEAF1A.cfg

```diff
@@ -1224,7 +1231,7 @@ router bgp 65101
       route-target export evpn 51:51
       router-id 192.168.255.3
       update wait-install
-      neighbor 10.255.251.1 peer group MLAG-IPv4-UNDERLAY-PEER
+      neighbor 10.255.251.1 peer group MLAG-IPv4-VRFS-PEER
```

Comment:
let's deactivate in one VRF the peering:

```
tenants:
  vrfs:
    - name:
       enable_mlag_ibgp_peering_vrfs: False
```

Reply:
Done!